backend/wine_map/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import generics
+from .models import Wine
+from rest_framework.pagination import LimitOffsetPagination
+from .serializers import WineSerializer
```

Comment:
According to [PEP-8](https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be,group%20of%20imports.) you should split your imports into three groups (from standard library, third-party imports and local app specific imports) with a blank line between each

```suggestion
from rest_framework import generics
from rest_framework.pagination import LimitOffsetPagination

from .models import Wine
from .serializers import WineSerializer
```

Reply:
you can use combination `control + option + O`